Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -9,7 +11,10 @@ public static class ArrayOperations
         /// <param name="array">Input array in a random order.</param>
         public static void Sort(int[] array)
         {
-            // ToDo: implement.
+            if (array != null && array.Length != 0)
+            {
+                Array.Sort(array);
```

Comment:
As with not using List methods, I do not believe Array methods are allowed here.